### PR TITLE
docker: add build of debs for Debian 11 Bullseye

### PR DESCRIPTION
**Pull request by btravouillon**
This commit adds a Dockerfile to build the debs for Debian 11. For the
build to succeed, update the file cobbler.spec to define the correct
package name in createrepo_pkg since Debian 11 moved to the C
implementation of createrepo.

Update the installation guide to provide the build and install command
for Debian 11. Also add Debian 11 to the Makefile and GH Action jobs.

### CODE REVIEW

File: .github/workflows/packaging.yml [hunk 1]
```diff
@@ -53,3 +53,11 @@ jobs:
         shell: 'script -q -e -c "bash {0}"'
         run: |
           ./docker/debs/build-and-install-debs.sh deb10 docker/debs/Debian_10/Debian10.dockerfile
+  build-debian11-debs:
```
Comment (SchoolGuy) [inline .github/workflows/packaging.yml:56]: I would love if you could please rename the Debian 10 job to `debian10` to be aligned to the new name.

**Review by SchoolGuy — COMMENTED** [review]
One small nitpick. Thanks for the addition!

File: .github/workflows/packaging.yml (hunk 1 above)
Comment (btravouillon) [reply .github/workflows/packaging.yml:56]: Done.

**Review by SchoolGuy — APPROVED** [review]
LGTM

**Comment by SchoolGuy** [comment]
@btravouillon Thanks a lot for the contribution!

**Comment by btravouillon** [comment]
@SchoolGuy you're welcome. Should I backport it to `release32` and `release33`?

**Comment by SchoolGuy** [comment]
@btravouillon Nope. We don't backport such things currently. If we want to have it, I do it with the next release!

**Pull request merged**